### ISSUE TITLE
abnormal pancreatic alpha cell mass

### ISSUE THREAD

**Issue by avucoh** [obophenotype/mammalian-phenotype-ontology#2759]
Why does this term exist for beta cells but not alpha cells? The complement should definitely exist in MPO.

>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)
abnormal pancreatic alpha cell mass

>2. Synonyms (e.g., absent spleen)

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

"anomaly in the total physical bulk or volume of a pancreatic alpha cell compared to the normal state"

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

MP:0005215

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

**Comment by sbello** [comment]
Thank you for the request. I would think the only reason this isn't in is we hadn't had a need or request for it yet. I'll add this soon.
Sue

**Comment by anna-anagnostop** [comment]
Added:

abnormal pancreatic alpha cell mass MP:0030587
SYN: abnormal glucagon-secreting cell mass
SYN: abnormal pancreatic A cell mass

decreased pancreatic alpha cell mass MP:0030588
SYN: reduced pancreatic alpha cell mass
SYN: decreased pancreatic A cell mass
SYN: decreased glucagon-secreting cell mass
SYN: reduced pancreatic A cell mass
SYN: reduced glucagon-secreting cell mass

increased pancreatic alpha cell mass MP:0030589
SYN: increased pancreatic A cell mass
SYN: increased glucagon-secreting cell mass
